docs/source/prereqs.md

```diff
@@ -172,10 +172,28 @@ Optional: Install the latest version of [jq](https://stedolan.github.io/jq/downl
 <!--- Indent entire section -->
 <div style="margin-left: 1.5em;">
 
-### Git
+### Docker
 
-Install the latest version of [git](https://git-scm.com/downloads) if it is not already installed.
-To use `Fabric binaries`, you will need to have the `uname` command available. You can get it as part of `Git` but beware that only the 64bit version is supported.
+Install the latest version of [Docker Desktop](https://docs.docker.com/get-docker/) if it is not already installed.
+
+### WSL2
+
+Both the fabric documentation and fabric samples rely heavily on a `bash` environment. The recommended
+path is to use WSL2 (Windows Subsystem for Linux version 2) to provide a native linux environment and then you can follow the linux pre-requisites section (excluding the linux docker pre-requisite as you already have docker Desktop) and install them into your WSL2 linux distribution.
+
+WSL2 may not be installed by default, you can check and install WSL2 by going into "Programs and Features", clicking on "Turn Windows features on or off" and ensuring that both "Windows Subsystem For Linux" and "Virtual Machine Platform" are ticked.
+
+Next you will need to install a linux distribution such as Ubuntu-20.04 and make sure it's set to using version 2 of WSL. Refer to https://docs.microsoft.com/en-us/windows/wsl/install for more information.
+
+Finally you need to ensure Docker Desktop has integration enabled for your distro so that the distro can interact with docker such as a bash command window. To do this, open the Docker Desktop gui and go into settings, select `Resources` and them `WSL Integration` and ensure the checkbox for enable integration is checked. You should then see your WSL2 linux distribution listed (if you don't then it's probably because it is still a WSL1 distribution and needs to be converted to WSL2) and you can then toggle the switch to enable integration for that distro. Refer to https://docs.docker.com/desktop/windows/wsl/ for more information
+
+### Microsoft VS Code (Optional)
+
+Microsoft VS Code provides an IDE that has tight integration with WSL2 distros using the Remote Development extension pack. This pack includes, among other things, the `Remote - WSL extension` and the `Remote - Containers` extension.
+
+### Git For Windows (Optional)
+
+Although not required, if you do decide to install git on windows and manage the fabric repos natively (as opposed to within WSL2 and it's git install) then make sure you configure git as follows
```

Comment:
- "... install GitHub on Windows and manage the Fabric repositories natively ... "
- " ... and its git install" 
- " ... install), then ... "
- " ... as follows:"